src/itrim.cpp

```diff
@@ -106,10 +106,9 @@ usage() {
         " ",
         "Reduce the number of replicas of a dataObject in iRODS by deleting some replicas.",
         "Nothing will be done if this is less than or equal to numCopies. The -n and",
-        "the -S options are used to specified which copies to delete. If neither of",
+        "the -S options are used to specify which replica to delete. If neither of",
```

Comment:
-S can target multiple replicas. Perhaps `replica` should be plural?

Reply:
agreed, will update.

Reply:
done.